docs/en/data_types/datetime64.md

```diff
@@ -0,0 +1,60 @@
+# DateTime64 {#data_type-datetime64}
+
+Stores a moment in time, like DateTime, but with 2 differences:
+* Data is stored in 'ticks', not seconds
+* Data is stored in `Int64` instead of `Int32` in DateTime
+
+Синтаксис:
+
+```sql
+DateTime64(precision, [timezone])
+```
+
+Tick size (precision) is defined as type parameter:
```

Comment:
we need to explain what does it mean first, not just by giving example

Reply:
Added